### ISSUE TITLE
Debranding should be easy

### ISSUE THREAD

**Issue by lisandi** [flectrahq/discussion#42]
It would be great if FlectraHQ would be complete White Labeled or at least easy to debrand without all the modules Yelizariev has already created for Odoo to do that tasks. I am sure that this would be also an important point for people to move from ODOO CE to FlectraHQ

**Comment by flectrahqadmin** [comment]
We slightly differ in our opinion over here, why to debrand something and present it as something else. Why the originality of application can be maintained. It's good that you can modify the look and feel of the application.

**Comment by lisandi** [comment]
No problem as long as it stays in the sourcecode like in any other CMS inkl Enterprise CMS TYPO3, Wordpress etc.
It is not about pretending to be something else it is simply a requirement that many customers wish to have as they don't like i.e. that on the frontend Odoo number#1 etc or then Flectra Number#2 etc shows up. Most of Templates you can buy at the Odoo store already take care of that incl the modules of Yelizariev. It is simply a bloat which is completely unnecessary if the branding stays where it belongs - in the code itself and isn't visible at first place in any frontend.

If this isn't the case then for example some of our customers already won't be willing to even giving it a try as it would be nothing else then what Odoo is already practicing and it would show the undemocratic way problems and things like branding would get handled. 

It is NOT to represent it as something else as THIS - to represent it as something else would probably happen when not ODOO but then FlectraHQ would get forked to get an unbranded version and I am sure this is for sure not your goal! I can only advise you to take all branding as well as all phoning back or mailing back or unallowed collecting of data without User consent out of FelctraHQ. 
A system which is good and which gets used by millions does not need any branding at all on the frontend - simply have a Look to Wordpress, TYPO3 has a note in the sourceode at the top of the page - That would be fullyOK also with FlectraHQ, it would respect the Freedoms of Free Open Source Software FLOSS.

**Comment by flectrahqadmin** [comment]
can we have a discussion on "places" where it is good to put the branding and where not to, so that it can be taken care. I know the "Home Page" bottom "Powered By Odoo the #1 Open Source eCommerce" is sort of growth hacking and back linking tactic, which helps in spreading the word but not much friendly.

**Comment by lisandi** [comment]
Yes of course we can and it is good to have. I have very good experiences with the branding of TYPO3. Absolute nothing appears in the frontend but have a look how much appears in a google search when looking for that here:

"This website is powered by TYPO3" - About 55,500 results
This website is powered by TYPO3  - About 163,000 results
"Powered by TYPO3" - About 116,000 results
Powered by TYPO3 - About 403,000 results
"This website is powered by TYPO3 - inspiring people to share!" - About 3,750 results
This website is powered by TYPO3 - inspiring people to share! - About 118,000 results

Check out the links!!! They will show you a very important fact! Especially by comparing
search for it as singe words or as a quote - Powered by TYPO3 or "Powered by TYPO3"
do the same for Odoo - Powered by ODOO or "Powered by ODOO"
now check out what the topics are on the first search page result pages!

In TYPO3 you won't find rarely a link about removing that "Powered by" text while nearly all Odoo search results are only about one single problem: How to remove that specific Line! 
 
TYPO3 Customers are PROUD to show that have a TYPO3 website - believe me - but none of them shows it the way Odoo does show it, but I would say 95% or pages keep the Part which is part of the header of each single webpage - check the source code. It is often even listed in another color and very easy to find right at the top! YES we use TYPO3, YES our company can afford a the Buggati of Open Source Enterprise CMSs, Yes we want to show this to the world, this is the message even our company gives to the world by keeping that small part of code in the source code of each single page we create. It gets automatically added always at the same place on top!

Even there is no text of link displayed by default on an empty page in the frontend of TYPO3 many people let others know by writing a line near the imprint and other places that it is TYPO3. They do that by themselves as their own decision because they don't get forced to have it by default!!!

Now do the same with Odoo 

"Powered by Odoo" - About 189,000 results
Powered by Odoo - About 400,000 results
"Powered by Odoo, the #1 Open Source eCommerce" - About 71,500 results
Powered by Odoo, the #1 Open Source eCommerce - About 74,900 results

Most of the links on the first google search result pages are actually about How to remove the "Powered by Odoo" stuff on the pages in the footer!!!

- not a great advertising for Odoo! - 

It shows that people hate that line and many simply don't know how to remove it! When it got removed, which is very easy using the debranding modules I mentioned previous Odoo won't be anymore recognized as Odoo at all! 

Do you want that for FlectraHQ? well then keep that Link and rename it and link it to FlectraHQ and people will start looking for something else for sure? But if you want people to be PROUD of using the best REAL No1 Open Source CMS which will stay Open Source and Free in LGPL or AGPL and will always be a full feature set etc. then you should put that link in a similar way to the source code like TYPO3 already does it and avoid actually all appearance in the frontend!

People using Odoo hate that Line to be honest. Until now all customers we talked to asked us if that specific line could be removed and if it is necessary to keep that line and especially the link back to Odoo. They simply don't like to be forced to have it. On the other hand customers where we had previous during development had used a template which removed that line automatically asked us if we would like to add a line about Odoo on a specific page - mostly near the imprint.

So simply be smart and don't be like a dictator telling people what they have to think, how they have to look like, what they have to do and are not allowed to do. Simply give them back their freedom and show them that you don't depend on that line to make your FlectrHQ a much better solution then Odoo has ever been. Give them back the 4 Freedoms of Free Open Source Software and don't force them to have it displayed branded in the frontend by default. i.e. simply build in a button where they can switch off all branding or make it more specific make a debranding section where all customers can decide themselves what they would like to show or not show. On the same form don't forget to place a button to support FlectraHQ with a link to the FlectraHQ PayPal. It is a great way to get what you actually need much more then only a link on the front-page ;-)

The more freedom you will give back to the people who use FlectraHQ, the more people will actually switch back to the only one real Free Open Source Full feature package which would be FlectraHQ and no more ODOO which gots limited as you described already since version 8 and has a hidden roadmap, nearly no transparency and linking back to them all the time by trying to get your users data to make marketing for themselves - lots of threads are about that topic in Odoo.

You won't find any thread about that Topic at TYPO3, as people are PROUD to use TYPO3. People are PROUD to display a Text and Link to TYPOP3 somewhere on the pages. Agencies and Developers are PROUD that they are able and qualified to offer TYPO3 Services. This feeling you will never get when you will force people to anything!!!

A good Leader empowers everyone who helps him to be the leader of a great product and he will put himself at last. Together they will be unbeatable - like TinyERP was actually at the beginning before the Tryton split! Always keep that mind.

You will have the same actually also in the proprietary sector as it is a part of leadership! Leadership you have to give to your community of FlectrsHQ!
https://www.facebook.com/Mindset2Succeed/videos/903594459793773/

Compare Apple and Microsoft!

Apple Users are PROUD to show their Apple
How often and where does Microsoft show something to make their people proud - well how can you be proud of viruses. ;-)

https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action/transcript

TRUST, LOYALTY
"The neocortex is responsible for all of our rational and analytical thought and language. The middle two sections make up our limbic brains, and our limbic brains are responsible for all of our feelings, like trust and loyalty. It's also responsible for all human behavior, all decision-making, and it has no capacity for language."

" The goal is not just to sell to people who need what you have; the goal is to sell to people who believe what you believe. The goal is not just to hire people who need a job; it's to hire people who believe what you believe."

Listen and read the transcript carefully again and again if you don't get it at the first time - I had the same problem and had to read it more then once! Simon Sinek is right!

" The people who believed in the Wright brothers' dream worked with them with blood and sweat and tears."

I would even go further then only debranding I would even rething your LGPL License at first place. LGPL means that you won't get anymore informed about improvements especially when you try to get also on the saas / cloud market what will be the future! Much better would be to get it back to AGPL and integrate the LGPL code of Odoo 11 into your newly AGPL licensed project.

Only this would force even ODOO S.A. to contribute to FlectraHQ if they would like to benefit from FlectraHQ too. You have right now the great option and check out the EU laws as they will enable you to do exactly that! I had posted a link concerning that already on the Odoo mailing list, while talking about NEWDOO.

I am sure that you will get a huge support from most of the Odoo Community if you will bring back what made ODOO great - the TINYERP Feeling and powerful Community Heart! Good Luck!

Sorry that this explanation is a bit longer but I think it is important to tell you why it is much more important to give people their FREEDOM to demonstrate that you BELIVE in the FlectraHQ community instead of dictating people your dreams!

TYPO3 has an important adding after the Powered by TYPO3. It is "Inspire to share"
Make this the concept of FLECTRAHQ and Flectra will be soon THE ERP used by major companies, organizations and people and all of them will be PROUD that you BELIEVE in them as they believe in the power of FlectraHQ!
